apps/etterlatte-beregning/src/main/kotlin/BeregningRepository.kt

```diff
@@ -0,0 +1,139 @@
+package no.nav.etterlatte
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import io.ktor.server.plugins.NotFoundException
+import kotliquery.Row
+import kotliquery.TransactionalSession
+import kotliquery.queryOf
+import kotliquery.sessionOf
+import kotliquery.using
+import no.nav.etterlatte.libs.common.beregning.Beregningsperiode
+import no.nav.etterlatte.libs.common.grunnlag.Metadata
+import no.nav.etterlatte.libs.common.objectMapper
+import no.nav.etterlatte.libs.common.toJson
+import no.nav.etterlatte.model.Beregning
+import java.util.*
+import javax.sql.DataSource
+
+interface BeregningRepository {
+    fun lagre(beregning: Beregning): Beregning
+    fun hent(behandlingId: UUID): Beregning
+}
+
+class BeregningRepositoryImpl(private val dataSource: DataSource) : BeregningRepository {
```

Comment:
Mangler tester for denne.

Reply:
Det kommer i denne branchen straks i en ny pr! https://jira.adeo.no/browse/EY-1147